docs/release_notes/release_notes-0.16.0.rst

```diff
@@ -67,6 +67,7 @@ Key Features and Updates
   * DOCS-#4552: Change default sphinx language to en to fix sphinx >= 5.0.0 build (#4553)
   * DOCS-#4628: Add to_parquet partial support notes (#4648)
   * DOCS-#4668: Set light theme for readthedocs page, remove theme switcher (#4669)
+  * DOCS-#4748: Apply the Triage label to new issues.   
```

Comment:
missing PR number in parentheses

Reply:
Yes, please correct this to be consistent.